troposphere/static/js/components/dashboard/InstanceHistoryList.react.js

```diff
@@ -47,6 +48,22 @@ define(function(require) {
       stores.InstanceHistoryStore.fetchMore();
     },
 
+    refreshHistory: function(){
+        stores.InstanceHistoryStore.fetchFirstPage();
+        stores.InstanceHistoryStore.lastUpdated = Date.now();
+        this.forceUpdate();
+    },
+
+    renderRefreshButton: function(){
+      
+      return (
+        <span className="pull-right instance-history">
+            <RefreshComponent onRefreshClick = {this.refreshHistory} timestamp = {stores.InstanceHistoryStore.lastUpdated} delay = {1000 * 60} />
```

Comment:
This is not totally important ... but my python-pep8 mind twitched on long lines like this... 

I'd be tempted to stack the attrs of that component element ... but 
```html
<RefreshComponent onRefreshClick = {this.refreshHistory} 
		timestamp = {stores.InstanceHistoryStore.lastUpdated} 
		delay = {1000 * 60} />
```